src/content/reference/react/useImperativeHandle.md

```diff
@@ -36,25 +36,25 @@ const MyInput = forwardRef(function MyInput(props, ref) {
 
 [See more examples below.](#usage)
 
-#### Parameters {/*parameters*/}
+#### Parameter {/*parameters*/}
 
-* `ref`: The `ref` you received as the second argument from the [`forwardRef` render function.](/reference/react/forwardRef#render-function)
+* `ref`: *ref* yang Anda terima sebagai argumen ke-dua dari [fungsi *render* `forwardRef`.](/reference/react/forwardRef#render-function)
 
-* `createHandle`: A function that takes no arguments and returns the ref handle you want to expose. That ref handle can have any type. Usually, you will return an object with the methods you want to expose.
+* `createHandle`: Sebuah fungsi yang tidak mengambil argumen dan mengembalikan penanganan ref yang ingin Anda ekspos. Penanganan ref tersebut dapat memiliki tipe *any*. Biasanya, Anda akan mengembalikan sebuah object dengan sekumpulan metode yang ingin Anda ekspos.
```

Comment:
```suggestion
* `createHandle`: Sebuah fungsi yang tidak mengambil argumen dan mengembalikan penanganan *ref* yang ingin Anda ekspos. Penanganan *ref* tersebut dapat memiliki tipe apapun. Biasanya, Anda akan mengembalikan sebuah objek dengan sekumpulan metode yang ingin Anda ekspos.
```

1. Please italicize `ref`
2. Please translate `object` to `object`. [Glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium#o)
3. `any` here is not in the context of `any` type literally. `createHandle` is a function, which shouldn't have an `any` type. It could be anything that is a subset of `function`.